apteryx.h

```diff
@@ -171,6 +178,17 @@ bool apteryx_set_full (const char *path, const char *value, uint64_t ts,
  */
 bool apteryx_set_tree_full (GNode *root, uint64_t ts, bool wait_for_completion);
 
+/**
+ * Set a tree of multiple values in Apteryx, with full options
+ * @param root pointer to the N-ary tree of nodes.
+ * @param ts monotonic timestamp to be compared to the paths last change time
+ * @param id is the pid of the process to not send a watch callback to
```

Comment:
How do we know the process id to not send the message to? Why are you picking on them only?